controller/features_test.go

```diff
@@ -2,21 +2,29 @@ package controller
 
 import (
 	"context"
+	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/fabric8-services/fabric8-toggles-service/app/test"
 	"github.com/goadesign/goa"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
 	"github.com/magiconair/properties/assert"
 	"testing"
 )
 
 func TestFeatures(t *testing.T) {
```

Comment:
I would rename this function to `TestListFeatures` so we know you're testing the `List` operation